Solutions/GoogleWorkspaceReports/Data Connectors/GWorkspaceReportsAPISentinelConnector/__init__.py

```diff
@@ -12,11 +11,20 @@
 import azure.functions as func
 import logging
 import os
+import json
 import time
 import re
 from .state_manager import StateManager
+from datetime import datetime, timedelta
 
-customer_id = os.environ['WorkspaceID'] 
+customer_id = os.environ['WorkspaceID']
+fetchDelay = os.getenv('FetchDelay',10)
+chunksize = 500
```

Comment:
500 is very less. LA can accept upto 25 MB, on an average it can be 10000 or more

Reply:
changed the value to be more realistic.